lib/Yield/Support/SugiyamaStyleGraphLayout.cpp

```diff
@@ -0,0 +1,11 @@
+/// \file SugiyamaStyleGraphLayout.cpp
+/// \brief
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "revng/Yield/Support/SugiyamaStyleGraphLayout.h"
+
+bool yield::sugiyama::layout(Graph &Graph, const Configuration &Configuration) {
+}
```

Comment:
Uh?

Reply:
This commit (similarly to the previous thread) only describes the interface. The actual functionality comes in the next commits.

Reply:
At least put a `revng_abort`. Doesn't this trigger warnings?

Reply:
`orc install revng@debug` didn't complain, so I guess not.

`revng_abort` works.

Reply:
Done.